tests/topotests/bgp_comm_list_match/r1/bgpd.conf

```diff
@@ -19,5 +20,9 @@ route-map r2 permit 20
  match ip address prefix-list p3
  set community 65001:3
 route-map r2 permit 30
+ match ip address prefix-list p4
+ set community 65001:10 65001:12 65001:13
```

Comment:
Can we test also from the other side with `match community ... any`?

Reply:
> Can we test also from the other side with `match community ... any`?

oups . forgot the r3 configuration. done